packages/discord.js/src/managers/GuildEmojiManager.js

```diff
@@ -51,13 +51,13 @@ class GuildEmojiManager extends BaseGuildEmojiManager {
    */
   async create({ attachment, name, roles, reason }) {
     attachment = await DataResolver.resolveImage(attachment);
-    if (!attachment) throw new TypeError(ErrorCodes.REQ_RESOURCE_TYPE);
+    if (!attachment) throw new TypeError(ErrorCodes.ReqResourceType);
```

Comment:
why this code uses short form?

Reply:
Unsure, as you can see, that's how it was before as well, so I'm probably not touching it.